### DISCUSSION TITLE
python hw1_3.py help

### DISCUSSION THREAD

**Discussion by Rodrigo-dev-bit** [1101-GS4719-C/Python-Community-GS4719-C#807] [Q&A]
**請問大家有遇到這種狀況嗎?**

我99乘法從2開始，每次 i+1都會print "==" ，到 i=9 就沒有"=='，但是 autograding沒給過?

![image](https://user-images.githubusercontent.com/55293456/196973546-2c788df8-25b3-493d-9996-90b8a2003f90.png)

**Comment by evanchou1204** [comment]
沒有欸，你是不是要貼給我們看程式碼才知道問題在哪？

**Comment by dabbie109303501** [comment]
你會不會是迴圈跑完之後忘記加到output裡面? 因為你的錯誤訊息寫assert "" == "2 * 2 = 4 ...."，感覺他前面的hw1_3.hw1_3()拿到的值是空字串

**Comment by ghost** [comment]
你可以複製你的程式到jupyter lab，讓他print()，看看是不是有哪裡排版上出錯。
